cli/asc.js

```diff
@@ -333,6 +333,13 @@ exports.main = function main(argv, options, callback) {
     }
   }
 
+  // If showConfig print args and exit
+  if (args.showConfig) {
+    // FIXME: not a proper C-like API
```

Comment:
In other places this comment indicates that something will be incompatible with the compiler compiled to Wasm (with asc remaining in JS), but here it doesn't need the comment because we are not calling into the non-cli `assemblyscript` library (that might at one point become a Wasm module), but the data is already exclusively on the JS side of things.